### PR TITLE
Add cli.color.{app,attachment,addon} methods

### PR DESCRIPTION
**Pull request by ransombriggs**

### CODE REVIEW

File: lib/color.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+let chalk = require('chalk');
+
+module.exports = Object.assign(chalk, {
+  app: chalk.cyan,
```
Comment (dickeyxxx) [inline lib/color.js]: this needs to be `app: s => chalk.cyan(s), ` in order for it to be properly disabled

**Comment by dickeyxxx** [comment]
fyi @bjeanes

**Comment by ransombriggs** [comment]
@dickeyxxx I added a commit which fixes the disabled color problem.

File: package.json [hunk 2]
```diff
@@ -19,7 +19,8 @@
     "jsdoc": "*",
     "jshint": "*",
     "mocha": "^2.2.4",
-    "nock": "^2.6.0"
+    "nock": "^2.6.0",
+    "lodash": "3.10.1"
```
Comment (dickeyxxx) [inline package.json]: this is not a dev dependency

**Comment by ransombriggs** [comment]
@dickeyxxx updated the lodash dependency

**Pull request merged**